File: PracticasProgramacion/Practica_07/python/auto.py
"""Clase Auto: precio y año. Equivalente a C++ Auto."""


class Auto:
    def __init__(self, precio: float = 0.0, anio: int = 0) -> None:
        self._precio = precio
        self._anio = anio

    def get_precio(self) -> float:
        return self._precio

    def set_precio(self, precio: float) -> None:
        self._precio = precio

    def get_anio(self) -> int:
        return self._anio

    def set_anio(self, anio: int) -> None:
        self._anio = anio

File: PracticasProgramacion/Practica_07/cpp/Auto.cpp
#include "Auto.h"

Auto::Auto() : precio_(0.0), anio_(0) {}

Auto::Auto(double precio, int anio) : precio_(precio), anio_(anio) {}

double Auto::getPrecio() const { return precio_; }
void Auto::setPrecio(double precio) { precio_ = precio; }
int Auto::getAnio() const { return anio_; }
void Auto::setAnio(int anio) { anio_ = anio; }
